docs/admin/access_control/service_accounts.mdx

```diff
@@ -0,0 +1,50 @@
+# Service Accounts
+
+Service accounts are specialized user accounts designed for automation, API integrations, and programmatic access to Sourcegraph, as opposed to using access tokens from regular users. Unlike regular user accounts, service accounts don't require an email address or password and cannot access the Sourcegraph UI. They are specifically intended for automated workflows, CI/CD pipelines, scripts, and other non-human access patterns. Service accounts also don't count towards the user limit and won't be part of any billing cycles.
+
+## Creating Service Accounts
```

Comment:
Two things we should call out clearly
1. Initially service accounts will have access to 0 repos and you must either explicitly assign repos (and tell how) OR
2. You have an option to mark the service account for access to ALL repos which  can be very useful for tool BUT  you must do so with care